packages/core/src/asset/mintDrop.ts

```diff
@@ -31,32 +61,42 @@ export async function mintDrop(
     minter: toAddress(address),
   })
 
-  const initialQuantity = await checkOwnership(
-    sdk,
-    drop.chainId,
-    drop.collectionAddress,
-    drop.id,
-    toAddress(address),
-  )
-
   onProgress?.({ type: 'TRANSACTION_SIGNATURE', payload: {} })
   const tx = await sendTransaction(signer, transaction)
 
   onProgress?.({
     type: 'TRANSACTION_PENDING',
     payload: { txHash: toTransactionHash(tx.hash) },
   })
-  await tx.wait()
+  const receipt = await tx.wait()
+  const transfers = [] as { tokenId: string }[]
+  for (const log of receipt.logs || []) {
+    try {
+      const parsed = transferInterface.parseLog(log)
+      if (parsed.name !== 'Transfer') continue
+      transfers.push({ tokenId: parsed.args.tokenId.toString() })
+    } catch (e) {}
+  }
```

Comment:
No idea how to test this and if this is correct implementation.
Just followed your comments. 😅